app/Http/Middleware/RequireTwoFactorAuthentication.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Pterodactyl\Http\Middleware;
+
+use Closure;
+use Krucas\Settings\Settings;
+use Prologue\Alerts\AlertsMessageBag;
+
+class RequireTwoFactorAuthentication
+{
+    const NOBODY = 0;
```

Comment:
I think these would work better as `LEVEL_NONE`, `LEVEL_ADMIN`, `LEVEL_ALL`.

Not entirely sure what the difference between users and everybody would be. Seems like a very strange edge-case to not require 2FA for admins, but require it for normal users.

Reply:
None = nobody obviously
Admin = administrators only
Users = admins + normal users only
All = admins + normal users + sub users + anybody else logged in

How does that sound?